website/docs/configuration/interpolation.html.md

```diff
@@ -355,6 +355,14 @@ The supported built-in functions are:
       `n` is the index or name of the subcapture. If using a regular expression,
       the syntax conforms to the [re2 regular expression syntax](https://github.com/google/re2/wiki/Syntax).
 
+  * `rsadecrypt(string, key)` - Decrypts `string` using RSA PKCS #1 v1.5.
+    The `string` may be base64-encoded or the raw cipher text. `key` must be an
```

Comment:
We've had some bad experiences in the past with functions that except either a raw value _or_ some encoded value, since it can cause surprising behavior if e.g. some input happens to _look_ like base64 even though it was supposed to be raw ciphertext.

I think it would be better to just always require base64 input here, and then users can use `base64encode` if they have some unencoded ciphertext to deal with. Note though that Terraform only guarantees safe passage for strings that are valid UTF-8, so it's generally unsafe to send non-text content through Terraform.